.git-blame-ignore-revs

```diff
@@ -1,56 +1,23 @@
-# PrmPkg: Apply uncrustify changes
-a298a84478053872ed9da660a75f182ce81b8ddc
-# UnitTestFrameworkPkg: Apply uncrustify changes
-7c0ad2c33810ead45b7919f8f8d0e282dae52e71
-# UefiPayloadPkg: Apply uncrustify changes
-e5efcf8be8a1bf59aa98875787475e3144ee4cef
-# UefiCpuPkg: Apply uncrustify changes
-053e878bfb5c9d5eca779789b62891add30b14ba
-# StandaloneMmPkg: Apply uncrustify changes
-91415a36ae7aaeabb2bbab3762f39544f9aed683
-# SourceLevelDebugPkg: Apply uncrustify changes
-c1e126b1196de75e0a4cda21e4551ea9bb05e059
-# SignedCapsulePkg: Apply uncrustify changes
-b87864896714cf3062a7bc6d577d8fbd62d105e5
-# ShellPkg: Apply uncrustify changes
-47d20b54f9a65b08aa602a1866c1b59a69088dfc
-# SecurityPkg: Apply uncrustify changes
-c411b485b63a671a1e276700cff025c73997233c
-# RedfishPkg: Apply uncrustify changes
-39de741e2dcb8f11e9b4438e37224797643d8451
-# PcAtChipsetPkg: Apply uncrustify changes
-5220bd211df890f2672c23c050082862cd1e82d6
-# OvmfPkg: Apply uncrustify changes
-ac0a286f4d747a4c6c603a7b225917293cbe1e9f
-# NetworkPkg: Apply uncrustify changes
-d1050b9dff1cace252aff86630bfdb59dff5f507
-# MdePkg: Apply uncrustify changes
-2f88bd3a1296c522317f1c21377876de63de5be7
-# MdeModulePkg: Apply uncrustify changes
-1436aea4d5707e672672a11bda72be2c63c936c3
+###################################################
+# Line Ending Only Changes                        #
+###################################################
+# Fixed new line endings being LF instead of CRLF
+53954a8aba9aeca7c784fad5bcce0b2015f330ea
+# Fix line endings in repo (LF -> CRLF)
+18949932d64eff23d78934a897ce7ff1ec1b2845
+# IntelFsp2WrapperPkg: Convert files to CRLF line ending
+ca467f68f4ce520ad0250678c91e9623a0195902
+# IntelFsp2Pkg: Convert files to CRLF line ending
+4aabee6980fb60aecc58e750560205abcf681236

```

Comment:
Are these old commits? Because line endings was done in one commit in 2405, right?

Reply:
Yeah, I've carried these forward from the previous branch since they either contain edk2 commits or will need adjusted when line endings are fixed when the plugin is added. In this case, I can update it since I already confirmed line endings at that point and throw in the uncrustify commit.

Reply:
Updated